zcutil/make-release.py

```diff
@@ -196,9 +196,8 @@ def verify_version(release, releaseprev, hotfix):
     expected = Version(
         releaseprev.major,
         releaseprev.minor,
-        releaseprev.patch,
+        releaseprev.patch + 1,
         releaseprev.betarc,
-        releaseprev.hotfix + 1 if releaseprev.hotfix else 1,
```

Comment:
This makes sense. Previously we assumed we'd never have more than 50 hotfixes in a row; now we assume we'll never have more than 100 hotfixes in a row. What a luxury!